tests/test_workflow_contracts.py

```
@@ -12,7 +12,8 @@
 class WorkflowContractTests(unittest.TestCase):
     """Protect the platform's workflow wrapper and reusable contract invariants."""
 
-    def test_reusable_mutation_workflows_do_not_reference_openai_api_key_lane(self) -> None:
+    def test_reusable_mutation_workflows_do_not_reference_openai_api_key_lane(
+        self, ) -> None:
```

Flake8 (7.3.0)

[error] continuation line with same indent as next logical line

(E125)